README.md

```diff
@@ -190,11 +197,13 @@ Currently we cover the following scenarios:
 
 - `npm run start` - start the application in production mode
 - `npm run dev` - start the application in development mode
-- `npm run register` - configure commercetools API Extension to point to your development server
-- `npm run unregister` - unregister commercetools API Extension
+- `npm run api-extension-add` - add commercetools API Extension pointing to your server (server url is taken from APP_URL environment variable)
+- `npm run api-extension-delete` - remove commercetools API Extension by Key value configured in COMMERCE_TOOLS_API_EXTENSION_KEY environment variable. Optionaly you can provide specific API Extension Id by `npm run api-extension-delete -- --id=xxx-xxx-xxx`
```

Comment:
Maybe consider adding more information and differences between ID and ATTR in registration and unregistration actions. And not to use the same attribute multiple times may it result in the removal of all connectors with such ATTR. Unless we recognise it as obvious. As I found `COMMERCE_TOOLS_API_EXTENSION_KEY` is used as ATTR each time in actions like `UPDATE`, 'ADD' but `DELETE` could deleting ATTR or ID as well.

Reply:
By default, we are using "Key" attribute, fresh installations do not need to use the option with ID. Removing the API Extension by ID is needed only when migrating from v1x to v2 and it's described in the migration section